variables.tf

```diff
@@ -1032,6 +1142,36 @@ variable "enable_flow_log" {
   default     = false
 }
 
+variable "enable_firewall_logs" {
+  description = "Whether or not to enable Network Firewall Logs"
+  type        = bool
+  default     = false
+}
+
+variable "firewall_log_cloudwatch_log_group_name_prefix" {
+  description = "Specifies the name prefix of Network Firewall Log Group for Network Firewall logs."
+  type        = string
+  default     = "/aws/network-firewall-log/"
+}
+
+variable "firewall_log_cloudwatch_log_group_retention_in_days" {
+  description = "Specifies the number of days you want to retain log events in the specified log group for Network Firewall logs."
+  type        = number
+  default     = null
+}
```

Comment:
```
  ~ resource "aws_cloudwatch_log_group" "firewall_log" {
        arn               = "arn:aws:logs:us-east-1:123456789123:log-group:/aws/network-firewall-log/vpc-022a76fc738dbd151-flow"
        id                = "/aws/network-firewall-log/vpc-00000000000000001-flow"
        name              = "/aws/network-firewall-log/vpc-00000000000000001-flow"
      ~ retention_in_days = 120 -> 0
        tags              = {
            "kubernetes.io/cluster/shared2-wallets-api-eks" = "shared"
        }
        tags_all          = {
            "kubernetes.io/cluster/shared2-wallets-api-eks" = "shared"
        }
    }
```
A default of `null` causes this var to not be marked as required in the docs automator, but will cause TF to constantly re-create this resource.  aws defaults to 120 days, update that to this value or something else.

Reply:
Updated default to 120